ShanghaiBloodSports/ProjectSettings/ProjectSettings.asset

```diff
@@ -586,6 +592,6 @@ PlayerSettings:
   projectName: Shanghai Blood Sports
   organizationId: loonman
   cloudEnabled: 0
-  enableNativePlatformBackendsForNewInputSystem: 0
+  enableNativePlatformBackendsForNewInputSystem: 1
```

Comment:
This file changed since it prompted me to fix the project settings for the new input system

Reply:
Kreygasm